src/main/java/com/esri/samples/portal/oauth/OAuthChallengeHandler.java

```diff
@@ -0,0 +1,117 @@
+/*

```

Comment:
@john0005 I think we should keep duplicates. In a real application you wouldn't duplicate, but in samples I think you want to see everything you need in one folder. Also, the sample viewer only looks in the sample folder and the fxml folder for its files list. There would be no easy rule to help it find the required files.